packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -911,6 +874,145 @@ class SliverReorderableListState extends State<SliverReorderableList> with Ticke
   }
 }
 
+/// An auto scroller that scrolls the [scrollable] if a drag gesture drag close
+/// to its edge.
+///
+/// The [ReorderableList] uses this scroller to auto scroll the scroll view when
+/// user drag item toward the edge of the scroll view.
+class _EdgeDraggingAutoScroller {
```

Comment:
This class will eventually moved to scrollable.dart once other global selection starts to use this class. Until then this class remains private.